fate-product-list.csv

```diff
@@ -1,302 +1,302 @@
-6/21/2024,Tianxia Accelerated,Vigilance Press,https://www.drivethrurpg.com/product/142131/Tianxia-Accelerated?affiliate_id=14…
-6/21/2024,"Tianxia: Blood, Silk & Jade",Vigilance Press,https://www.drivethrurpg.com/product/126883/Tianxia-Blood-Silk--Jade?affiliate_…
-6/21/2024,"Tianxia: Spirits, Beasts & Spells",Vigilance Press,http://www.drivethrurpg.com/product/188855/Tianxia-Spirits-Beasts--Spells?affil…
-6/21/2024,"Tianxia: War, Iron & Stone",Vigilance Press,http://www.drivethrurpg.com/product/202396/Tianxia-War-Iron--Stone?affiliate_id…
-6/21/2024,Aces in Space,Vogt & Vriends,https://www.drivethrurpg.com/product/317497/Aces-In-Space?affiliate_id=144937
-6/21/2024,Oubliette Second Edition,Voidspiral Entertainment,http://www.drivethrurpg.com/product/204460/Oubliette-Second-Edition?affiliate_i…
-6/21/2024,Diaspora,VSCA Publishing,http://drivethrurpg.com/product/79933/Diaspora?affiliate_id=144937
-6/21/2024,Wearing the Cape: The Roleplaying Game,Wearing the Cape Productions,https://www.drivethrurpg.com/product/210012/Wearing-the-Cape-The-Roleplaying-Ga…
-6/21/2024,"Crestfallen, A Bronze Age Fantasy RPG",White Rose Games,http://www.drivethrurpg.com/product/176630/Crestfallen-RPG?affiliate_id=144937
-6/21/2024,Once Upon a Crime,Wooden Vampire Games,http://www.drivethrurpg.com/product/171284/Once-Upon-a-Crime?affiliate_id=144937
-6/21/2024,Age of Arthur,Wordplay Games,http://www.drivethrurpg.com/product/111752?affiliate_id=144937
-6/21/2024,Time of the Wolves,Wordplay Games,http://www.drivethrurpg.com/product/149507?affiliate_id=144937
-6/21/2024,Strays!,Wordsmith Games LLC,http://www.drivethrurpg.com/product/169261/Strays?affiliate_id=144937
-6/21/2024,Mecha vs Kaiju: A Sci-Fi Anime RPG for Fate Core,WrightWerx,https://www.drivethrurpg.com/product/141809/Mecha-vs-Kaiju-Fate-Core?affiliate_…
-6/21/2024,Mecha vs Kaiju: Big Book of Kaiju - Land (Fate Core),WrightWerx,http://www.drivethrurpg.com/product/183921/Mecha-vs-Kaiju-Big-Book-of-Kaiju--La…
-6/21/2024,Mecha vs Kaiju: Big Book of Kaiju - Strange (Fate Core),WrightWerx,http://www.drivethrurpg.com/product/216360/Mecha-vs-Kaiju-Big-Book-of-Kaiju--St…
-6/21/2024,Mecha vs Kaiju: Kaijupunk 202X,WrightWerx,https://www.drivethrurpg.com/product/335925/Mecha-vs-Kaiju-Kaijupunk-202X?affil…
-6/21/2024,Mecha vs Kaiju: Sensational Sentai Squad GO!,WrightWerx,http://www.rpgnow.com/product/208158/Mecha-vs-Kaiju-Sensational-Sentai-Squad-GO…
-6/21/2024,Mecha vs Kaiju: Steam and Steel - A Fate Steampunk Supplement,WrightWerx,http://drivethrurpg.com/product/183915/Mecha-vs-Kaiju-Steam-and-Steel--A-Fate-S…
-6/21/2024,Michtim for Fate,Zev Mir,https://grimogre.itch.io/michtim-fate
\ No newline at end of file
+Product title,Publisher Name,Link to the product
+Aether Sea • A World of Adventure for Fate Core,Evil Hat Productions,http://rpg.drivethrustuff.com/product/139872/Aether-Sea--A-World-of-Adventure-for-Fate-Core?affiliate_id=144937

```

Comment:
Probably out of scope for this MR, but wondering: does the CSV ordering affect the order of products / publishers on the final page? What's the intended ordering?

I could see an argument that with a product that has expansions we'd like the core book first - but that's not guaranteed to sort first, depending on what the names are like.

Reply:
This PR meets the issues immediate needs, so I would recommend that we merge this and open a separate issue to discuss the way that the list should be structured and used.